### PR TITLE
open-cluster-management/backlog#18387 ki subnet req

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: clusters/create_cluster_on_prem.adoc [hunk 1]
```diff
@@ -58,6 +58,8 @@ The locations of the available hosts that meet the requirements for this cluster
 +
 *Note:* This value must match the name that you used to create the DNS records listed in the prerequisites section. If not provided, the DNS must be pre-configured so that `test.apps.` resolves correctly.
 
+. Select a subnet from the list of available subnets. 
+
```
Comment (jelkosz) [inline clusters/create_cluster_on_prem.adoc]: actually, reading it again, its already documented above: 
`After your hosts finish binding and display a `Bound` status, select a subnet for your cluster from the list of options for the bound host`

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -160,6 +160,12 @@ When observability is enabled or disabled on {ocp-short} managed clusters that a
 [#cluster-management-issues]
 == Cluster management known issues
 
+[#subnet-required-on-prem-clust-create]
+=== Selecting a subnet is required when creating an on-premises cluster
+//2.4:18387
+
+When you create an on-premises cluster using the {product-title-short} console, you must select an available subnet for your cluster. It is not marked as a required field.
```
Comment (jelkosz) [inline release_notes/known_issues.adoc:167]: Its correct, but FYI this is planned to be fixed in 2.5 where we will add way more validations and will mark those fields as mandatory. But yes, currently this is correct.

File: clusters/create_cluster_on_prem.adoc (hunk 1 above)
Comment (cadawson) [reply clusters/create_cluster_on_prem.adoc]: Good point. That should work.

Comment (cadawson) [reply clusters/create_cluster_on_prem.adoc]: Removed the duplicate.

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (cadawson) [reply release_notes/known_issues.adoc:167]: We will have the opportunity to scrub this before 2.5 releases. We can remove it at that time.

**Comment by jelkosz** [comment]
lgtm

**Pull request merged**